pom.xml

```diff
@@ -99,6 +102,38 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.1.1</version>
+                <configuration>
+                    <configLocation>${checkstyle.path}/checkstyle.xml</configLocation>
+                    <includeTestResources>false</includeTestResources>
+                    <encoding>UTF-8</encoding>
+                    <consoleOutput>true</consoleOutput>
+                    <failsOnError>true</failsOnError>
+                    <linkXRef>false</linkXRef>
+                    <propertyExpansion>samedir=${checkstyle.path}</propertyExpansion>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>validate</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+<!--                This doesn't work for some reason so had to reference the project directly. -->
```

Comment:
Is there a user story to solve that?

Reply:
Currently no. I just kept it here for reference...

Reply:
I removed it now.